test/performance/scale_from_zero_test.go

```diff
@@ -0,0 +1,192 @@
+// +build performance
+
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package performance
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	pkgTest "github.com/knative/pkg/test"
+	"github.com/knative/pkg/test/logging"
+	"github.com/knative/test-infra/shared/testgrid"
+	"golang.org/x/sync/errgroup"
+	"k8s.io/api/extensions/v1beta1"
+
+	"github.com/knative/serving/test"
+)
+
+const (
+	serviceName                      = "perftest-scalefromzero"
+	ScaleFromZeroAvgTestGridProperty = "perf_ScaleFromZero_Average"
```

Comment:
If I understood correctly, it's easiest to keep this at "perf_latency", because that will show all numbers on one dashboard, right @srinivashegde86?